### PR TITLE
Add latest version of UniCoq and Mtac2

### PR DESCRIPTION
**Pull request by MSoegtropIMC**
This command adds opam packages for the latest version of UniCoq and Mtac2.

This are plain unmodified tag versions.

### CODE REVIEW

File: released/packages/coq-mtac2/coq-mtac2.1.3+8.12/opam [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "ocaml"
+  "coq" {>= "8.12.0" & < "8.13~"}
+  "coq-unicoq" {>= "1.5" & < "2~"}
+]
+synopsis: "Mtac2: Typed Tactics for Coq"
```
Comment (clarus) [inline released/packages/coq-mtac2/coq-mtac2.1.3+8.12/opam:20]:
```suggestion
synopsis: "Mtac2: Typed Tactics for Coq"
tags: [
  "logpath:Mtac2"
  "date:2020-08-19"
]
```

File: released/packages/coq-unicoq/coq-unicoq.1.5+8.12/opam [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "ocaml"
+  "coq" {>= "8.12.0" & < "8.13~"}
+]
+synopsis: "An enhanced unification algorithm for Coq"
```
Comment (clarus) [inline released/packages/coq-unicoq/coq-unicoq.1.5+8.12/opam:19]:
```suggestion
synopsis: "An enhanced unification algorithm for Coq"
tags: [
  "logpath:Unicoq"
  "date:2020-08-12"
]
```

**Comment by clarus** [comment]
I wonder if we should add the tags. I added some even if in the previous versions there were no tags.

**Comment by MSoegtropIMC** [comment]
> I wonder if we should add the tags. I added some even if in the previous versions there were no tags.

Thanks - indeed a good idea.

**Pull request merged**